### ISSUE TITLE
Remove usernames in favor of email/phone number 

### ISSUE THREAD

**Issue by mackfoggia** [PSU-CapstoneBrownHope/BrownHopeAidApp#220]
Nothing in this app requires users to have a username, having them log in with email or phone number and a password instead would be cleaner and easier for the user. The problem is that the username is fairly well entangled in the code, so adjusting this would take a bit of work.

**Comment by baloveless** [comment]
could we have the username field be assigned to the email without removing all the things that rely on it? Or is the username used as a key for some of the airtable searches as well?

**Comment by mackfoggia** [comment]
Yeah, it is used in at least a few of the air table calls in the back end.
